jenkins-scripts/docker/gz-abichecker.bash

```diff
@@ -51,29 +51,29 @@ else
   export ABI_JOB_HEADER_PREFIX=${ABI_JOB_SOFTWARE_NAME/[ignz]*-/}[0-9]*
 fi
 
-# check if NEED_C17_COMPILER should be set
+# check if NEED_C17_COMPILER should be set. Only needed for Bionic so only care about Citadel
+# versions
 if [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-gazebo" ]] || \
   [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-physics" ]] || \
   [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-sensors" ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-common"    && ${GZ_NAME_PREFIX_MAJOR_VERSION} -ge 3 ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-gui"       && ${GZ_NAME_PREFIX_MAJOR_VERSION} -ge 1 ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-math"      && ${GZ_NAME_PREFIX_MAJOR_VERSION} -ge 6 ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-msgs"      && ${GZ_NAME_PREFIX_MAJOR_VERSION} -ge 3 ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-plugin"    && ${GZ_NAME_PREFIX_MAJOR_VERSION} -ge 1 ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-rendering" && ${GZ_NAME_PREFIX_MAJOR_VERSION} -ge 1 ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-transport" && ${GZ_NAME_PREFIX_MAJOR_VERSION} -ge 6 ]]
+  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-common" ]] || \
+  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-gui" ]] || \
+  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-math" ]] || \
+  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-msgs" ]] || \
+  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-plugin" ]] || \
+  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-rendering" ]] || \
+  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-transport" ]]
 then
   export NEED_C17_COMPILER=true
 fi
 
 # check if OGRE-2.2 include paths are needed
-if [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-gazebo"   && ${GZ_NAME_PREFIX_MAJOR_VERSION} -eq 6 ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-gui"       && ${GZ_NAME_PREFIX_MAJOR_VERSION} -eq 6 ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-launch"    && ${GZ_NAME_PREFIX_MAJOR_VERSION} -eq 5 ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-sensors"   && ${GZ_NAME_PREFIX_MAJOR_VERSION} -eq 6 ]] || \
-  [[ "${ABI_JOB_SOFTWARE_NAME}" = "ign-rendering" && ${GZ_NAME_PREFIX_MAJOR_VERSION} -eq 6 ]]
+if [[ "${ABI_JOB_SOFTWARE_NAME}" = "gz-sim"      && ${GZ_NAME_PREFIX_MAJOR_VERSION} -eq 6 ]] || \
```

Comment:
Since these packages are in Fortress, I'm not sure if we should make the `ign` -> `gz` rename here.

Reply:
Let try to get rid of as much as ign as possible.

Reply:
Okay, we probably have to revert this then after https://github.com/gazebo-tooling/release-tools/pull/1040